db/migrate/20181001131632_add_conversion_host_id_to_miq_request_tasks.rb

```diff
@@ -0,0 +1,46 @@
+class AddConversionHostIdToMiqRequestTasks < ActiveRecord::Migration[5.0]
+  class MiqRequestTask < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    serialize :options, Hash
+    belongs_to :conversion_host, :class_name => "AddConversionHostIdToMiqRequestTasks::ConversionHost"
+  end
+
+  class ConversionHost < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    belongs_to :resource, :polymorphic => true
+  end
+
+  class Host < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    add_column :miq_request_tasks, :conversion_host_id, :bigint
+
+    MiqRequestTask.where(:type => 'ServiceTemplateTransformationPlanTask').each do |task|
+      host_id = task.options.delete(:transformation_host_id)
+      next unless host_id
+      host = Host.find_by(:id => host_id)
```

Comment:
`Host.find(host_id)`

Reply:
We are expecting that this may be `nil` so `find_by` was chosen on purpose here to avoid the exception.

Reply:
Right above it is `next unless host_id`, so it will never be nil...

Reply:
If the host was deleted it will be nil.  Since this isn't a proper relation now there is not `:dependent => nullify`